src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-size: 16px;
+  font-family: Avenir-Book, Helvetica, sans-serif;
+  font-style: normal;
+}
+
+.form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search {
+  font-family: Avenir-Book, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-color: #fff;
+  border-radius: 4px;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.smallSearch {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 400;
+
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-Heavy, Helvetica, sans-serif;
+  color: #3d4e61;
+  outline: none;
+}
+
+.search::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
```

Comment:
You can move the color to the [global variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). It will be much better to follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) rules

Reply:
ok